src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -24,16 +24,27 @@
 using System.Linq;
 using System.Reflection;
 using System.Threading.Tasks;
+#if NETCOREAPP2_1
+
+using SonarScanner.MSBuild.AnalysisWarning;
+
+#endif
 using SonarScanner.MSBuild.Common;
 using SonarScanner.MSBuild.Common.Interfaces;
 
 namespace SonarScanner.MSBuild
 {
     public class BootstrapperClass
     {
-        public const int ErrorCode = 1;
-        public const int SuccessCode = 0;
+        private const int ErrorCode = 1;
+        private const int SuccessCode = 0;
+#if NETCOREAPP2_1
+
+        private const string WarningMessage = "From [Date], we will no longer release new Scanner for .NET versions that target .NET Core 2.1." +
+            " If you are using the .NET Core Global Tool you will need to use a supported .NET runtime environment." +
+            " For more information see https://community.sonarsource.com/t/net-runtime-support-for-the-sonarscanner-for-net/54684";
```

Comment:
Short-end URL reduces noise
```suggestion
            " For more information see https://community.sonarsource.com/t/54684";
```